templates/log4j2.xml.erb

```diff
@@ -138,7 +138,7 @@ if ! httpAppenders.empty?
 <%- end -%>
         </Logger>
 <%- if @herp -%>
-        <Logger name="org.openrepose.herp.post.filter" level="info">
+        <Logger name="org.openrepose.herp.post.filter" level="info" additivity="false">
```

Comment:
based on what this does, why wouldnt we be making this a parameter to the module? Is the default true? if so this will change the behavior.